README.md

```diff
@@ -1,2 +1,4 @@
 # longhorn-tests
-Various test cases for longhorn
+A setup and a teardown script for setting up machines for various longhorn tests
```

Comment:
This is still no instruction on how to use it. Need a README.md(and not in the root directory) which people can follow exactly then successful run the test.